Makefile

```diff
@@ -20,7 +20,7 @@ check: generate
 	cargo clippy --all-targets --features "${ALL_FEATURES}" -- -D clippy::all
 
 unit-test: generate
-	cargo test --all --no-default-features
+	cargo nextest run --all --no-default-features
```

Comment:
Suggest to also add command for installing `nextest` in `Makefile`. Otherwise running unit tests locally would fail.

Rest LGTM~

Reply:
add the installation guid for `nextest` in README, just for simplicity fellow what `tiup` did, which also a prerequisite for run IT.